src/riak_kv_vnode.erl

```diff
@@ -110,6 +110,21 @@
                 reqid :: term(),
                 target :: pid()}).
 
+-record(counter_state, {
```

Comment:
Ok, but it's not a purposeless counter, it has a specific role. Is there a good name for its role?